native/src/main/java/io/ballerina/stdlib/grpc/builder/syntaxtree/utils/MessageUtils.java

```diff
@@ -124,6 +124,9 @@ private static Type getMessageType(Message message) {
                 case "Duration":
                     messageRecord.addCustomFieldWithDefaultValue("time:Seconds", field.getFieldName(), "0.0d");
                     break;
+                case "Struct":
+                    messageRecord.addCustomFieldWithDefaultValue("map<anydata>", field.getFieldName(), "{}");
```

Comment:
Shall we create a new method calleed `addMapFieldWithDefaultValue` similar to `addMapField`[[1]](https://github.com/ballerina-platform/module-ballerina-grpc/blob/d269ad779461978fe548d2817a8103cc8b181a12/native/src/main/java/io/ballerina/stdlib/grpc/builder/syntaxtree/components/Record.java#L90).



[1] https://github.com/ballerina-platform/module-ballerina-grpc/blob/d269ad779461978fe548d2817a8103cc8b181a12/native/src/main/java/io/ballerina/stdlib/grpc/builder/syntaxtree/components/Record.java#L90

Reply:
Changed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/368/commits/67eb9b0783f2bd76dfa9565bd36c53e66aec9d88